elements/boron.md

```diff
@@ -7,3 +7,4 @@ Symbol: B<br/>
 Atomic Number: 5<br/>
 
 Reason I chose this element: the name makes me smile
+Small change to boron

```

Comment:
This is actually not any information about Boron! Please add information about Boron.